wordpress-mobile/WordPress-Android
# Extract preview action to view model and test it

This PR extracts the functionality to show a preview from the Adapter to the ViewModel & fragment. This allows us to test the functionality

To test:
- Go to the media picker
- Long press on a photo
- Check that the preview is shown

PR submission checklist:
